src/lib/types.ts

```diff
@@ -0,0 +1,107 @@
+export type TCategory = {
```

Comment:
do you think we can generate the typings from the graphql? you can check this `https://the-guild.dev/graphql/codegen`